### PR TITLE
Update per_component_reboot_test.go

### PR DESCRIPTION
**Pull request by sezhang2**
Validate the standby controller is online after the reboot

Test passed on hardware platform

### CODE REVIEW

File: feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go [hunk 1]
```diff
@@ -100,12 +100,22 @@ func TestStandbyControllerCardReboot(t *testing.T) {
...
+	startReboot := time.Now()
 	rebootResponse, err := gnoiClient.System().Reboot(context.Background(), rebootSubComponentRequest)
 	if err != nil {
 		t.Fatalf("Failed to perform component reboot with unexpected err: %v", err)
 	}
 	t.Logf("gnoiClient.System().Reboot() response: %v, err: %v", rebootResponse, err)
 
+	watch := dut.Telemetry().Component(rpStandby).RedundantRole().Watch(
+		t, 10*time.Minute, func(val *telemetry.QualifiedE_PlatformTypes_ComponentRedundantRole) bool {
+			return val.IsPresent()
```
Comment (xw-g) [inline feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go:112]: are you checking for if the value exist, or if the value is the one expected?

Comment (sezhang2) [reply feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go:112]: Check if the value exists/IsPresent.

File: feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go [hunk 2]
```diff
@@ -230,6 +240,13 @@ func findComponentsByType(t *testing.T, dut *ondatra.DUTDevice, cType telemetry.
 func findStandbyRP(t *testing.T, dut *ondatra.DUTDevice, supervisors []string) (string, string) {
 	var activeRP, standbyRP string
 	for _, supervisor := range supervisors {
+		watch := dut.Telemetry().Component(supervisor).RedundantRole().Watch(
+			t, 5*time.Minute, func(val *telemetry.QualifiedE_PlatformTypes_ComponentRedundantRole) bool {
+				return val.IsPresent()
+			})
+		if val, ok := watch.Await(t); !ok {
+			t.Fatalf("DUT did not reach target state: got %v", val.String())
```
Comment (xw-g) [inline feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go:248]: nit: same

File: feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go [hunk 3]
```diff
@@ -100,12 +100,22 @@ func TestStandbyControllerCardReboot(t *testing.T) {
...
 		t.Fatalf("Failed to perform component reboot with unexpected err: %v", err)
 	}
 	t.Logf("gnoiClient.System().Reboot() response: %v, err: %v", rebootResponse, err)
 
+	watch := dut.Telemetry().Component(rpStandby).RedundantRole().Watch(
+		t, 10*time.Minute, func(val *telemetry.QualifiedE_PlatformTypes_ComponentRedundantRole) bool {
+			return val.IsPresent()
+		})
+	if val, ok := watch.Await(t); !ok {
+		t.Fatalf("DUT did not reach target state: got %v", val.String())
```
[the remark this answers is not in the document]
Comment (sezhang2) [reply feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go]: Done

**Review by xw-g — APPROVED** [review]

File: feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go (hunk 2 above)
Comment (sezhang2) [reply feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go]: done

**Review by xw-g — APPROVED** [review]

**Pull request merged**